field_encrypt.module

```diff
@@ -85,7 +85,6 @@ function field_encrypt_form_alter(&$form, FormStateInterface $form_state, $form_
       );
 
       // We add functions to process the form when it is saved.
-      $form['actions']['submit']['#submit'][] = 'field_encrypt_update_encryption_form_submit';
```

Comment:
Hmmmm... where/how is this link shown? Would this ever be exposed to a site visitor?